### PR TITLE
Ability to set font for JSyntaxTextArea

### PR DESCRIPTION
**Pull request by d0k1**

### CODE REVIEW

**Comment by milamberspace** [comment]
Thanks for your PR.
Some comments:
* That would be better to use Constants (final public static)  for properties strings inside .java files (like jsyntaxtextarea.font, etc.) Particularly with the property key is use into several java files.

* The change inside the file JMeterUtils.java seems not necessary?

* Probably the best place inside the jmeter.properties for this new properties is the section Look&feel (start at line ~110)

**Comment by pmouawad** [comment]
Hi,
Thanks for PR.
I created https://bz.apache.org/bugzilla/show_bug.cgi?id=58933 for follow up.

@milamberspace  I let you handle this.
@d0k1 , if you intend to submit more PRs, it is good to create a bugzilla for it.
Thanks

**Comment by d0k1** [comment]
Hi! thanks for comments.
I'll make a bugzilla issue by myself, next time.
Could you tell me what should I do next?

**Comment by milamberspace** [comment]
@d0k1: You can modify your PR to include the comments above directly on your local branch, commit, squash your commits into one commit, push with --force option to update your PR on Github.
After ping me (on github or bugzilla) to allow me to include your code in JMeter source repository.

File: src/core/org/apache/jmeter/gui/util/JSyntaxTextArea.java [hunk 1]
```diff
@@ -90,6 +95,9 @@ public JSyntaxTextArea(int rows, int cols, boolean disableUndo) {
         super.setLineWrap(LINE_WRAP);
         super.setWrapStyleWord(WRAP_STYLE_WORD);
         this.disableUndo = disableUndo;
+        if(USER_FONT_FAMILY!=null) {
+            setFont(new Font(USER_FONT_FAMILY, Font.PLAIN, USER_FONT_SIZE));
```
Comment (milamberspace) [inline src/core/org/apache/jmeter/gui/util/JSyntaxTextArea.java]: @d0k1 your PR have a bug: when the jsyntaxtextarea.font.size property is defined and jsyntaxtextarea.font.family is *not* defined, the size is not change on the textarea.
Probably, you must define the default value to "JSyntaxTextArea.getDefaultFont().getName()" the USER_FONT_FAMILY constants, and remove this if condition.

**Comment by milamberspace** [comment]
@d0k1 don't forget to ping me on github after the update of your PR.

**Comment by d0k1** [comment]
@milamberspace Thanks for pointing me a bug. I've fixed it, squashed all commits into one.

**Comment by milamberspace** [comment]
@d0k1 Thanks for your improvement patch. Committed on jmeter repo, sorry for long delay for commit.

**Pull request closed without merging**